pubspec.yaml

```diff
@@ -46,6 +46,54 @@ dependency_overrides:
   path_provider: ^2.0.0-nullsafety.1
 
 flutter:
+  deferred-components:
```

Comment:
By making crane deferred, what happens on platforms that do not support deferred components?

Reply:
Nothing, they behave/build as normal. The tooling will handle assets declared here as regular assets defined in the `assets` section. Deferred builds only happen when building release appbundles.

Reply:
This is the error I am always getting. how to fix the deferred-components? this is my error:

[gallery] flutter create --ios-language swift --android-language kotlin .
Error detected in pubspec.yaml:
Unexpected child "deferred-components" found under "flutter".
Please correct the pubspec.yaml file at .\pubspec.yaml
exit code 1

--

[gallery] flutter create --ios-language swift --android-language kotlin .
Error detected in pubspec.yaml:
Unexpected child "deferred-components" found under "flutter".
Please correct the pubspec.yaml file at .\pubspec.yaml
exit code 1

--

[gallery] flutter pub get
Error detected in pubspec.yaml:
Unexpected child "deferred-components" found under "flutter".
Please correct the pubspec.yaml file at D:\Flutter\gallery\pubspec.yaml
exit code 1

Reply:
It looks like your tool is outdated? Try `flutter upgrade` after switching to beta or master channel. The new tooling has support for deferred components and the pubspec validator should recognize the new setting.